draft-ietf-jsonpath-base.md

```diff
@@ -289,23 +293,25 @@ JSONPath expressions are applied to a JSON value, the *argument*.
 Within the JSONPath expression, the abstract name `$` is used to refer
 to the *root node* of the argument, i.e., to the argument as a whole.
 
-JSONPath expressions can use the *dot notation*
+JSONPath expressions primarily use the *bracket notation*
 
 ~~~~
-$.store.book[0].title
+$['store']['book'][0]['title']
 ~~~~
 
-or the more general *bracket notation*
+but may also use the shorthand *dot notation*
 
 ~~~~
-$['store']['book'][0]['title']
+$.store.book[0].title
 ~~~~
 
 to build paths that are input to a JSONPath implementation.
+Generally, the dot notation is preferred for most applications.
```

Comment:
I'm not sure why we are expressing such a preference. It would be helpful to explain why this is the case.